pkg/apis/storage/types.go

```diff
@@ -363,11 +363,11 @@ type CSIDriverSpec struct {
 	// most one token is empty string. To receive a new token after expiry,
 	// RequiresRepublish can be used to trigger NodePublishVolume periodically.
 	//
-	// This is an alpha feature and only available when the
```

Comment:
Can you keep the comment but change it to beta?

Reply:
beta is enabled by default right?

Reply:
It is enabled by default, but it could be disabled by a cluster operator. I think we mainly want to inform users that this is still a beta feature so they shouldn't depend on the API/feature being stable yet.

Reply:
makes sense, done.